js/viz/sankey/layout.js

```diff
@@ -196,7 +196,7 @@ let layout = {
             rectsOfCascade.forEach(nodeRect => {
                 let nodeTitle = nodeRect._name,
                     rectFrom = this._findRectByName(rects, nodeTitle),
-                    linksFromNode = links.filter(link => { return link[0] === nodeTitle }); // all outgoing links from the node
+                    linksFromNode = links.filter(link => { return link[0] === nodeTitle; }); // all outgoing links from the node
```

Comment:
@roman-simionov @AlekseyMartynov  @nightskylark 

`omitLastInOneLineBlock: true`?

Reply:
It will allow:
```
expr();
expr();
expr() // which is a mess (IMO)
```

Reply:
We usually put a semicolon in such blocks see a [sample](https://github.com/DevExpress/DevExtreme/blob/18_2/testing/tests/DevExpress.viz.gauges/tracker.tests.js#L97)

In this case, it will better the following notation links.filter(link =>link[0] === nodeTitle);